### PR TITLE
Crew creation

### PR DESCRIPTION
**Pull request by vvdam**
Create a crew when you create an account as a 'Soundwaver'.

### CODE REVIEW

**Review by Dezau1995 — APPROVED** [review]

File: client/src/components/Modal/ModalCreateAccount.jsx [hunk 1]
```diff
@@ -57,7 +57,16 @@ function ModalCreateAccount({ closeModalCreateAccount, role }) {
         }
       );
       if (response.ok) {
-        navigate(`/user-profile/`);
+        if (role === "client") {
+          navigate(`/user-profile/`);
+        } else if (role === "crew") {
+          const { users, token } = await response.json();
+          setAuth({ isLogged: true, users, token });
+          console.info(users);
```
Comment (Alexandre78R) [inline client/src/components/Modal/ModalCreateAccount.jsx]: delete console.info

File: client/src/components/Modal/ModalCreateAccount.jsx [hunk 2]
```diff
@@ -57,7 +57,16 @@ function ModalCreateAccount({ closeModalCreateAccount, role }) {
...
+
+          console.info(token);
```
Comment (Alexandre78R) [inline client/src/components/Modal/ModalCreateAccount.jsx]: delete console.info

File: client/src/pages/CrewCreation/CrewCreation.jsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+        const result = await response.json();
+        setUsername(result.name);
+        setDescription(result.description);
+        setImage(result.image);
+      } else {
+        setErrors({ update: "Échec de la creatoin du compte" });
+      }
+      if (response.ok) {
+        const crew2 = await response.json();
+        console.info(crew2);
```
Comment (Alexandre78R) [inline client/src/pages/CrewCreation/CrewCreation.jsx]: delete console.info

File: server/database/models/CrewRepository.js [hunk 4]
```diff
@@ -12,6 +12,24 @@ class CrewRepository extends AbstractRepository {
     );
     return events;
   }
+
+  // async addIdOwner(userId) {
```
Comment (Alexandre78R) [inline server/database/models/CrewRepository.js]: delete code dead please

**Review by Alexandre78R — CHANGES_REQUESTED** [review]

**Review by VinchC — APPROVED** [review]

**Review by VinchC — APPROVED** [review]

**Pull request merged**